### ISSUE TITLE
Witness Plugin Softfork for Discounted Account Creation

### ISSUE THREAD

**Issue by mvandeberg** [steemit/steem#1772]
The witness plugin needs to reject account creations that do not have the requisite stake or PoW using the stake-based and PoW-based markets #1767 #1768

If an account creation does not have enough stake or PoW, the remaining costs (as a percentage of the market price) can be paid via STEEM as a fee.

**Comment by youkaicountry** [comment]
Blocked on RC Bandwidth implementation.

**Comment by theoreticalbts** [comment]
Possibly duplicate of #2648.

**Comment by mvandeberg** [comment]
I agree, finishing #2648 completes this issue.
Closing as a duplicate.
